src/pages/Menu/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import './MenuItem.css';
+
+class MenuItem extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isExpanded: false,
+      inputValue: 0
+    };
+  }
+
+  toggleExpand = (e) => {
+    e.stopPropagation();
+    this.setState(prevState => ({
+      isExpanded: !prevState.isExpanded
+    }));
+  };
+
+  handleSubmit = () => {
+    const valueToAdd = Number(this.state.inputValue);
+    this.props.handleAddition(valueToAdd);
```

Comment:
Having this component state and call handleAddition looks quite weird. It seems to me you have 2 places with information about amount of ordered items. Not a really good idea.